editor/src/utils/fake-localforage.ts

```diff
@@ -0,0 +1,19 @@
+class FakeLocalForage {
```

Comment:
We do something similar to this via a jest mock, except that backs it with an object to actually store and retrieve the results. Maybe it's worth doing that to prevent confusing issues later down the line? <https://github.com/concrete-utopia/utopia/blob/2a66237225cf36e7f6cf259010e13d03316dacfa/editor/src/components/editor/persistence/persistence.spec.ts#L154-L167>

Reply:
That code was the inspiration for mine, for benchmarking I think we don't want it to do anything because it could skew results in unexpected ways.